core/src/dird/backup.cc

```diff
@@ -849,7 +849,14 @@ void UpdateBootstrapFile(JobControlRecord* jcr)
       }
       if (VolParams) { free(VolParams); }
       if (got_pipe) {
-        CloseBpipe(bpipe);
+        int status = CloseBpipe(bpipe);
+        if (status) {
+          BErrNo err;
+          Jmsg(jcr, M_ERROR, 0,
```

Comment:
the next part of the code only reports an M_ERROR when not being able to open the file, shouldn't that also mark the job as error?

Reply:
You are right, the error handling should be like when the file cannot be opened. 

Then we also need to add the `jcr->setJobStatus(JS_ErrorTerminated);` to make sure the job is terminated.

M_ERROR is counted as warning. Only if too many such M_ERRORs appear, the job will be cancelled. Otherwise it will be marked as "OK- with warnings".